format/spec.md

```diff
@@ -702,6 +703,49 @@ Blob metadata is a struct with the following fields:
 | _optional_ | _optional_ | **`properties`** | `map<string, string>` | Additional properties associated with the statistic. Subset of Blob properties in the Puffin file. |
 
 
+#### Partition statistics
+
+Partition statistics files are based on [Partition Statistics file spec](#partition-statistics-file). 
+Partition statistics are not required for reading or planning and readers may ignore them.
+Each table snapshot may be associated with at most one partition statistic file.
+A writer can optionally write the partition statistics file during each write operation. If the statistics file is written for the specific snapshot,
+it must be registered in the table metadata file to be considered as a valid statistics file for the reader.
+
+Partition statistics files metadata within `partition-statistics` table metadata field is a struct with the following fields:
```

Comment:
Nit: does not make too much sense, does this suffice?

`Partition statistics files contain a struct `partition-statistics' with the following fields`

Reply:
I copied from puffin statistics file statements few lines above. 

changed it to

`partition-statistics` field of table metadata is an optional list of struct with the following fields: